SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -226,22 +257,16 @@ bool GenerateLaunchConfig(const FString& LaunchConfigPath, const FSpatialLaunchC
 			Writer->WriteObjectEnd(); // World section end
 			Writer->WriteObjectStart(TEXT("load_balancing")); // Load balancing section begin
 				Writer->WriteArrayStart("layer_configurations");
-				for (const auto& Worker : InWorkers)
+				if (InWorker.WorkerLoadBalancing != nullptr)
 				{
-					if (Worker.Value.WorkerLoadBalancing != nullptr)
-					{
-						WriteLoadbalancingSection(Writer, Worker.Key, *Worker.Value.WorkerLoadBalancing, Worker.Value.bManualWorkerConnectionOnly);
-					}
+					WriteLoadbalancingSection(Writer, SpatialConstants::DefaultServerWorkerType, *InWorker.WorkerLoadBalancing, InWorker.bManualWorkerConnectionOnly);
 				}
 				Writer->WriteArrayEnd();
-			Writer->WriteObjectEnd(); // Load balancing section end
-			Writer->WriteArrayStart(TEXT("workers")); // Workers section begin
-				for (const auto& Worker : InWorkers)
+				Writer->WriteObjectEnd(); // Load balancing section end
+				Writer->WriteArrayStart(TEXT("workers")); // Workers section begin
```

Comment:
these should be unindented to match the indentation of where the load_balancing starts